app/ui-react/syndesis/src/modules/integrations/pages/create/configure/addConnection/SelectActionPage.tsx

```diff
@@ -72,7 +76,7 @@ export class SelectActionPage extends React.Component {
                       header={<IntegrationCreatorBreadcrumbs step={3} />}
                       sidebar={
                         <WithIntegrationHelpers>
-                          {({ getSteps }) => {
+                          {({}) => {
```

Comment:
well this can be nuked now!

Reply:
well, lemme get on that :-)  and all those others

Reply:
right, so even `WithIntegrationHelpers` isn't needed in these cases, of course silly me...